### ISSUE TITLE
So is this project dead?

### ISSUE THREAD

**Issue by ghost** [rpgboss/rpgboss#542]
I would love people to still update this, even if it's just the community, as it has so much potential. This and Open RPG Maker (discontinued in 2014) had an amazing amount of potential. Where did it all go?

**Comment by tetra184** [comment]
Pretty much the maker has no time no more I'm thinking about maybe doing a
reboot

**Comment by AntumDeluge** [comment]
I would love to either have it re-written in another language, or at least output native game binaries that don't require Java.

**Comment by tetra184** [comment]
That would require alot of tinkering I will try keep you informed about a
reboot if get o e going

**Comment by ghost** [comment]
Thanks for the replies! I don't know Java or any language that would be good for this, but I hope someone can do this in the future! Such a great engine!

**Comment by tetra184** [comment]
Could be a good reason to learn and we will need then just coders we will
need promoters some pr and so on there's always room for people

**Comment by AntumDeluge** [comment]
I've messed with the source code a little. Even made a couple pull requests. But I am not familiar with Scala. I have done some Java though.

**Comment by ghost** [comment]
Could you potentially write it in C? Or a form of C? I know someone who's good at C#. Me? I'm not that good at programming, but am at video editing

**Comment by tetra184** [comment]
All help is welcome we could write it in a y language we like potentially
but obviously it would starting a blank slate and it's I'm not a code
genius just done some bits and pieces I can't see why it wouldn't be done
in c sharp but doing it java ensures it's always multi platform

**Comment by ghost** [comment]
Ok, I know someone who does Java, but he's doing university and probably won't be able to help :/
Hope others can try and do it!

**Comment by tetra184** [comment]
We will find some one I'm gonna make a fork soon and we can maybe start
something or we can try start from scratch

**Comment by ghost** [comment]
Awesome. Hope it goes well

**Comment by EgyLynx** [comment]
Hmm... Where begin? 
What size tiles? 32*32 or 64*64 . is 1024*1024 max at these groups? i meen tiles at 32*32 is 32*32 64*64 is 16*16? 
Isn´t first that? What size charatest/persons? 24*32, 32*32, or what?

**Comment by ghost** [comment]
What are you on about

**Comment by tetra184** [comment]
Tioe sizes are on the wiki

**Comment by tetra184** [comment]
Most comply to rpg maker standards

**Comment by EgyLynx** [comment]
OH? I think that all knows clear that i talking that remake, NOT that stopped think...

Hmm, why i can´t EDIT that?

**Comment by EgyLynx** [comment]
> Most comply to rpg maker standards

Hmm... I TALKING future NOT PAST.

But seems at :(

**Comment by ghost** [comment]
I'm learning JavaScript. Might pop into Java at some point. Really want to delve into this engine.

**Comment by tetra184** [comment]
Right so I've had a bad few months but I am going to get around to making a
fork and potentially a new site for this project and see if we can get a
good community around it and try and keep it going if any body is
interested in helping out that would be awesome

**Comment by ghost** [comment]
Definitely! Would love to get this project standing on it's own two legs!

**Comment by tetra184** [comment]
Sweet then I'll get something sorted asap and get back to you all and we
can see what we can do about getting website and some programmers behind us

**Comment by frothyejaculator** [comment]
This project died the moment the second kid popped out of the womb.
